gluon/custom_import.py

```diff
@@ -67,6 +67,13 @@ def custom_importer(name, globals=None, locals=None, fromlist=None, level=-1):
         import_tb = None
         try:
             try:
+                if ('gluon' not in name and not name.startswith('.') and
```

Comment:
I this if name does not start with 'gluon.' and does not start with '.' should try name = 'gluon.'+name but we should not check for "gluon" in name.